react/.storybook/preview.js

```diff
@@ -1,58 +1,92 @@
-import { addDecorator, addParameters } from '@storybook/react';
-import { withA11y } from '@storybook/addon-a11y';
-import { withInfo } from '@storybook/addon-info';
+import React from 'react';
+import ReactDOMServer from 'react-dom/server';
+import {
+  Title,
+  Subtitle,
+  Primary,
+  ArgsTable,
+  Heading,
+  ComponentsTable,
+  Stories,
+  PRIMARY_STORY,
+  CURRENT_SELECTION,
+  SourceState,
+  DocsContext,
+  Canvas, Story
+} from '@storybook/addon-docs/blocks';
+import { ActionBar, Source } from '@storybook/components';
+import prettier from 'prettier/standalone';
+import parserHtml from 'prettier/parser-html';
+import { Prism as SyntaxHighlighter } from 'react-syntax-highlighter';
+import { okaidia } from 'react-syntax-highlighter/dist/esm/styles/prism';
 import '../src/index.scss';
 
 const storyKindOrder = [
   'about', // storyKindOrder.indexOf -1 follow alphabetical order
   'brand', // storyKindOrder.indexOf -1 follow alphabetical order
   'dataviz', // storyKindOrder.indexOf -1 follow alphabetical order
-  'forms|atoms',
-  'forms|molecules',
-  'forms|organisms',
-  'forms|context',
+  'forms',
```

Comment:
ideally, atoms should be ordered before molecules in storybook, not sure why it's not by default

Reply:
![Screen Shot 2020-09-10 at 1 11 05 PM](https://user-images.githubusercontent.com/5789411/92770640-24eece80-f368-11ea-848e-a0aaa9cc9bf4.png)

Reply:
Atoms are before molecules, but not within the Forms category. I can fix this in the PR for all the forms changes.